Clinical-Genomics/cg
# accepts only OF rml 1604:9

This PR accepts only OF rml 1604:9

How to test:
0. install branch both on clinical-api and cgweb stage on clinical-db
1. login to OP
1. import an 1604:9 OF with some samples

Expected outcome:
The samples are imported to be further handled in the OP
